packages/nouns-webapp/src/pages/Vote/index.tsx

```diff
@@ -134,6 +152,17 @@ const VotePage = ({
     };
   })();
 
+  const desctructiveStateButtonAction = isCancellable ? <Trans>Cancel</Trans> : '';
+  const desctructiveStateAction = (() => {
```

Comment:
`desctructive` -> `destructive`

Reply:
corrected in commit
https://github.com/EricDevito/nouns-monorepo/commit/493711d4c9607714c3f3d538f457cd38a7bb169e